README.md

```diff
@@ -1 +1,48 @@
-# project-2
\ No newline at end of file
+<img src="./public/assets/bb3Blue.png">
+
+# Business Blueprint
+
+Business Blueprint allows any small business to keep track of their day to day operations
+
+
+## Utilization
+
+This application will allow a business owner to:
+- Add customers to their database
+- Add inventory items to their database
+- Create and keep track of customer invoices and sales
+- Search for invoices, sales orders, customer and employee information
+
+
+## Technologies Used
+
+- Node.js
+- Express Web Server
+- MySQL Database
+- Sequelize ORM
+- Auth0
+
+
+## Deployment
+
+- https://github.com/lalliere/project-2
+- https://dashboard.heroku.com/apps/bcs-project-2
+
```

Comment:
just put the broken one bc that one is useless to people outside of the group...there's at least a chance we get ours working

Reply:
very well! I will fix!